main/settings/src/main/scala/sbt/Reference.scala

```diff
@@ -78,7 +78,7 @@ object Reference {
       case LocalRootProject    => "{<this>}<root>"
       case LocalProject(id)    => "{<this>}" + id
       case RootProject(uri)    => "{" + uri + " }<root>"
-      case ProjectRef(uri, id) => "{" + uri + "}" + id
+      case ProjectRef(uri, id) => s"""ProjectRef(uri("$uri"),"$id")"""
```

Comment:
I am not interested in having a new function for this. What I would like is that the implementation of the new function becomes the implementation of display, and that it is used by **all** its current users. You can read my previous messages to understand why I'm asking that. There is no reason to show by default the scala code for project reference, and we better only show just the project reference and we achieve your primary concern: copy-paste ability.